frontend/my-app/src/pages/teamPage.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import CreateMeeting from '../components/CreateMeeting';
+
+import MeetingBarChart from '../components/MeetingBarChart';
+
+import MemberTable from '../components/MemberTable';
+import DonutChart from '../components/DonutChart';
+
+import {Card} from 'react-bootstrap'
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+
+import './teamPage.css';
+
+const TeamPage = (props) => {
+  return (
+    <div>
+
```

Comment:
Any reason for these random three spaces?

Also, may be worth using a linter (prettier) in the future to standardize code styling